docs/sql-guide/functions/function-identifier.md

```diff
@@ -0,0 +1,48 @@
+---
+title: IDENTIFIER()
+layout: default
+parent: Functions
+grand_parent: SQL guide
+---
+
+# IDENTIFIER() function
+
+`IDENTIFIER()` function generates unique IDs for the provided index.
+
+## Syntax
+
+```
+identifier(expr)
+```
+
+## Arguments
+
+| Argument | Description |
+|---|---|
+| `expr` | `index` name as `string`. |
+
+## Returns
+
+| Data type | Value |
+|---|---|
+| `id` | `unique ID created. |
```

Comment:
The value also matches the data type of the ID column.

So this can be clarified as follows:

```
| Data type | Value |
|---|---|
| `ID` or `STRING` | A valid, unique value is created for each row of data |
```